cardano-api/src/Cardano/Api/TxMetadata.hs

```diff
@@ -15,6 +15,8 @@ module Cardano.Api.TxMetadata (
     -- * Validating metadata
     validateTxMetadata,
     TxMetadataRangeError (..),
+    txMetadataTextStringMaxByteLength,
+    txMetadataByteStringMaxLength,
```

Comment:
I don't see these used anywhere outside this module and it's not exported from `Cardano.Api`.

What's the reason for this change?

Reply:
Indeed, oversight from some previous approach and subsequent refactor. Removed.